lib/secure-boot-lib.robot

```diff
@@ -4,6 +4,21 @@ Documentation       Collection of keywords related to UEFI Secure Boot
 Resource            ../keywords.robot
 
 
+*** Variables ***
+${GOOD_KEYS_URL}=           https://cloud.3mdeb.com/index.php/s/SjM5dQGji4XDAni/download/good_keys.img
+${GOOD_KEYS_NAME}=          good_keys.img
+${GOOD_KEYS_SHA256}=        13de737ba50c8d14a88aaf5314a938fb6826e18ba8f337470ee490b17dd6bea8
+${NOT_SIGNED_URL}=          https://cloud.3mdeb.com/index.php/s/zmJXxGG4piGB2Me/download/not_signed.img
+${NOT_SIGNED_NAME}=         not_signed.img
+${NOT_SIGNED_SHA256}=       15dc0a250b73c3132b1d7c5f8e81f00cc34d899c3ddecbb838a8cd0b66c4f608
+${BAD_KEYS_URL}=            https://cloud.3mdeb.com/index.php/s/BJPbSqRH6NdbRym/download/bad_keys.img
+${BAD_KEYS_NAME}=           bad_keys.img
+${BAD_KEYS_SHA256}=         6da92bd97d4b4ca645fa98dcdfdc0c6876959e5b815a36f1f7759bc5463e7b19
+${BAD_FORMAT_URL}=          https://cloud.3mdeb.com/index.php/s/AsBnATiHTZQ6jae/download/bad_format.img
+${BAD_FORMAT_NAME}=         bad_format.img
+${BAD_FORMAT_SHA256}=       59d17bc120dfd0f2e6948a2bfdbdf5fb06eddcb44f9a053a8e7b8f677e21858c
```

Comment:
@macpijan I'm not huge fan of relying on cloud.3mdeb.com to be honest it would be much better to rely on 3mdeb.com subdirectories or maybe some dedicated repository. From what I see footprint of cloud.3mdeb.com grow and I don't know the reason why. We already faced mass 404 when links were broken. I know we rely our business of DES on that, but in long run I think this is wrong strategy. Why can't we use dl.3mdeb.com ?

Reply:
> Why can't we use dl.3mdeb.com ?

It has limited access policy, and stores some critical data from other websites, so mistakes may be costly.

Perhaps the repo for blobs such as these would make sense.

Reply:
@macpijan we can use other server or repo as you suggest.

Reply:
We have more places like this. I have created an issue for improving that: https://github.com/Dasharo/open-source-firmware-validation/issues/121
In this MR we will focus on the SB functionality.